deploy.sh

```diff
@@ -73,15 +73,15 @@ push_helm_charts() {
 
 	mkdir -p "$CHARTDIR/csi-charts/docs/$PACKAGE"
 	cp -R "./charts/ceph-csi-$PACKAGE" "$CHARTDIR/csi-charts/docs/$PACKAGE"
-	pushd "$CHARTDIR/csi-charts/docs/$PACKAGE" >/dev/null
+	pushd "$CHARTDIR/csi-charts/docs/$PACKAGE" >/dev/null || exit
```

Comment:
Would you not need to fail the script when an error occurs? Use `exit 1` instead.

Or use `set -e` in the top of the script to exit on any error in the script.

Reply:
added `set -e` now dropping exit 1